Tests/Opc.Ua.Client.Tests/ClientTest.cs

```diff
@@ -657,6 +657,14 @@ public async Task ReconnectSessionOnAlternateChannelWithSavedSessionSecrets(stri
                 var result = session1.ReadValue(VariableIds.Server_ServerStatus, typeof(ServerStatusDataType));
                 Assert.NotNull(result);
             }
+
+            session1.DeleteSubscriptionsOnClose = true;
+            session1.Close(1000);
+            Utils.SilentDispose(session1);
+
+            session2.DeleteSubscriptionsOnClose = true;
+            session2.Close(1000);
+            Utils.SilentDispose(session2);
         }
```

Comment:
noticed in the test builds there were leaking 'opc.https' sessions that still do keep alive